csharp-intermediate-topics/ConstVsReadonlyInCSharp/ConstVsReadonlyInCSharp/MathCalculator.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ConstVsReadonlyInCSharp
+{
+    public class MathCalculator
```

Comment:
There is no instance data and none of the methods rely on instance data. This class and it's methods should be static

Reply:
I use it as an example of how to assign a complex type to a readonly field

Reply:
Missed that use case. My bad on that. I was looking at it trying to figure out why it wasn't static.